fe/fe-core/src/main/java/org/apache/doris/load/loadv2/SparkEtlJobHandler.java

```diff
@@ -255,6 +295,17 @@ public void killEtlJob(SparkAppHandle handle, String appId, long loadJobId, Spar
         }
     }
 
+    private SparkRepository.SparkArchive prepareDppArchive(SparkResource resource) throws LoadException {
+        // remote archive
+        SparkRepository.SparkArchive archive = null;
+        SparkRepository repository = resource.getRemoteRepository();
```

Comment:
Each SparkResource has and only has one SparkRepository.
So why not just call `resource. prepareDppArchive()` and return `SparkRepository.SparkArchive`?